Assets/Scripts/Gameplay/Action/ConcreteActions/ChaseAction.cs

```diff
@@ -1,9 +1,13 @@
+using System;
 using Unity.Multiplayer.Samples.BossRoom.Server;
+using Unity.Multiplayer.Samples.BossRoom.Visual;
 using Unity.Netcode;
 using UnityEngine;
 
 namespace Unity.Multiplayer.Samples.BossRoom.Actions
 {
+
+    [Serializable]
```

Comment:
no create menu attribute for this one?

Reply:
and other actions actually? why do we have it for certain actions and not others?

Reply:
fixed!